05/app.js

```diff
@@ -9,27 +9,102 @@ const stats = {
 
 /* tutaj umieść swój kod */
 
+const pElList = document.querySelectorAll('p');
+
+
+// Zliczanie paragrafów
+
+const counterParagraphs = function (e) {
+
+    if (this.dataset.id === 'p1') {
+        stats.paragraphs.p1 += 1;
+    }
+
+    if (this.dataset.id === 'p2') {
+        if (stats.paragraphs.p2 === undefined) {
+            stats.paragraphs.p2 = 0;
+            stats.paragraphs.p2 += 1;
+        } else {
+            stats.paragraphs.p2 += 1;
+        }
+    }
+
+    if (this.dataset.id === 'p3') {
+        if (stats.paragraphs.p3 === undefined) {
+            stats.paragraphs.p3 = 0;
+            stats.paragraphs.p3 += 1;
+        } else {
+            stats.paragraphs.p3 += 1;
+        }
+    }
```

Comment:
Spróbuj to zrobić bardziej automatycznie.
Jeśli będzie 100 takich paragrafów to wszystkie wypiszesz?
Wykorzystaj coś takiego `stats.paragraphs[this.dataset.id]`, analogicznie w linkach ;)